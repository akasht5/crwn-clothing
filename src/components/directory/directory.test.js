import React from 'react'
import { shallow } from 'enzyme'
import { store } from '../../redux/store'
import { Provider } from 'react-redux'
import Directory from './directory.component'

describe('<Directory />',() => {
    const mockSections = [];

    let wrapper = shallow(
    <Provider store={store}>
        <Directory mockSections={mockSections} />
    </Provider>
    );

    it('Check for snapshot testing',() => {
        expect(wrapper).toMatchSnapshot();
    })
})