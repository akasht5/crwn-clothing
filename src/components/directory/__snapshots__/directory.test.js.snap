// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Directory /> Check for snapshot testing 1`] = `ShallowWrapper {}`;
